src/topwidget.cpp

```diff
@@ -10,27 +10,21 @@
 TopWidget::TopWidget(QWidget *parent) :
     QToolBar(parent)
 {
-    mp_historyBackAction = new QAction(this);
-    mp_historyBackAction->setIcon(QIcon(":/icons/back.svg"));
-    mp_historyBackAction->setText(gt("back"));
-    mp_historyBackAction->setToolTip(gt("back"));
-    mp_historyBackAction->setShortcut(QKeySequence(Qt::ALT + Qt::Key_Left));
-    mp_historyBackAction->setEnabled(false);
-    connect(mp_historyBackAction, &QAction::triggered, [](){
+    auto app = KiwixApp::instance();
+
+    QAction *back = app->getAction(KiwixApp::HistoryBackAction);
+    connect(back, &QAction::triggered, [](){
         KiwixApp::instance()->getTabWidget()->triggerWebPageAction(QWebEnginePage::Back);
     });
-    addAction(mp_historyBackAction);
-    widgetForAction(mp_historyBackAction)->setObjectName("backButton");
-    mp_historyForwardAction = new QAction(this);
-    mp_historyForwardAction->setIcon(QIcon(":/icons/forward.svg"));
-    mp_historyForwardAction->setText(gt("forward"));
-    mp_historyForwardAction->setToolTip(gt("forward"));
-    mp_historyForwardAction->setShortcut(QKeySequence(Qt::ALT + Qt::Key_Right));
-    mp_historyForwardAction->setEnabled(false);
-    connect(mp_historyForwardAction, &QAction::triggered, [](){
+    addAction(back);
+    widgetForAction(back)->setObjectName("backButton");
```

Comment:
I add to do some git bisect to find why we add this (and not for the forwardButton). (https://github.com/kiwix/kiwix-desktop/commit/e9420e10c72884e77092a10694c2fbe43be70e43)

Can you add a small comment `// For css style` at the end of the line please ?

Reply:
done